tests/unit/_internal/bento/test_bentos.py

```diff
@@ -15,11 +15,11 @@ def test_create_simplebento(tmpdir):
         version="1.0",
         build_ctx="./simplebento",
         additional_models=[],
-        include=["*.py", "config.json", "somefile", "*dir*"],
+        include=["*.py", "config.json", "somefile", "*dir*", ".bentomlignore"],
         exclude=[
             "*.storage",
             "/somefile",
-        ],  # + anything specified in .bentoml_ignore file
+        ],  # + anything specified in .bentomlignore file
```

Comment:
not related to this PR - one thing I noticed when building bento with 1.0 was lots of `.pyc` files and `__pycache__` directory was included in the final Bento. Do you think we should add those to the default exclude list?